pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -940,6 +943,40 @@ type NomadSDConfig struct {
 	EnableHTTP2 *bool `json:"enableHTTP2,omitempty"`
 }
 
+// Service of the targets to retrieve. Must be `VPS` or `DedicatedServer`.
+// +kubebuilder:validation:Enum=VPS;DedicatedServer
+type OVHService string
+
+const (
+	VPS             OVHService = "VPS"
+	DedicatedServer OVHService = "DedicatedServer"
+)
+
+// OVHCloudSDConfig configurations allow retrieving scrape targets from OVHcloud's dedicated servers and VPS using their API.
+// See https://prometheus.io/docs/prometheus/latest/configuration/configuration/#ovhcloud_sd_config
+// +k8s:openapi-gen=true
+type OVHCloudSDConfig struct {
+	// Access key to use. https://api.ovh.com.
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	ApplicationKey string `json:"applicationKey"`
+	// +required
+	ApplicationSecret corev1.SecretKeySelector `json:"applicationSecret"`
+	// +required
+	ConsumerKey corev1.SecretKeySelector `json:"consumerKey"`
+	// Service of the targets to retrieve. Must be `VPS` or `DedicatedServer`.
+	// +kubebuilder:validation:Enum=VPS;DedicatedServer
+	// +optional
+	Service *OVHService `json:"service,omitempty"`
```

Comment:
If I read the Prometheus code properly, service is mandatory field.
```suggestion
	Service OVHService `json:"service"`
```

Reply:
My bad. Got it. 
Done.